app/components/Map/map_component.jsx

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import { View } from "react-native";
+import HeaderComponent from "../shared/header/header_component";
+import ToolbarComponent from "../shared/toolbar/toolbar_component";
+import { mapStyles } from "./_map";
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import DropDownMenuProps from '@material-ui/core/MenuItem';
+import { Avatar, Card, Title, Paragraph } from 'react-native-paper';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Button from "@material-ui/core/Button";
+import { ClickAwayListener } from '@material-ui/core';
+
+const LeftContent = props => <Avatar.Icon {...props} icon="folder" />
+
+
+class MapComponent extends React.Component {
+
+    constructor() {
+        super();
+        this.address = {
+            addr : [{address: "East Oakland Youth Development Center", mile: "500 ft", hour: "2 hrs"},
+                {address: "Black Organizing Project", mile: "0.2 mi", hour: "2 hrs"},
+                {address: "East Oakland Collective", mile: "0.7 mi", hour: "2 hrs"},
+                {address: "Oakland YMCA", mile: "1 mi", hour: "3 hrs"},
+                {address: "Tri-City Health Center", mile: "1.5 mi", hour: "2 hrs"}
+            ]
+        }
+
+        this.state = {
+
+            resource:false
+        }
+
+    //     this.resource = false;
+    //
+      this.updateResourceToTrue = this.updateResourceToTrue.bind(this)
+        this.updateResourceToFalse = this.updateResourceToFalse.bind(this)
+    }
+
+    updateResourceToTrue = () => {
+            this.setState({
+            resource:true});
+        }
+
+    updateResourceToFalse = () => {
+        this.setState({
+            resource:false});
+    }
+
+
```

Comment:
Everything looks good. If I had to nit-pick, the spacing in this file is kinda jank.